# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ezmlm/head_n.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./auto_bin.c.do.orig	2005-03-15 21:54:27.889464304 -0400
+++ ./auto_bin.c.do	2005-03-15 21:54:27.919459744 -0400
@@ -1,3 +1,3 @@
 dependon auto-str conf-bin
-formake './auto-str auto_bin `head -1 conf-bin` > auto_bin.c'
-./auto-str auto_bin `head -1 conf-bin`
+formake './auto-str auto_bin `head -n 1 conf-bin` > auto_bin.c'
+./auto-str auto_bin `head -n 1 conf-bin`
--- ./auto_qmail.c.do.orig	2005-03-15 21:54:27.935457312 -0400
+++ ./auto_qmail.c.do	2005-03-15 21:54:27.937457008 -0400
@@ -1,3 +1,3 @@
 dependon auto-str conf-qmail
-formake './auto-str auto_qmail `head -1 conf-qmail` > auto_qmail.c'
-./auto-str auto_qmail `head -1 conf-qmail`
+formake './auto-str auto_qmail `head -n 1 conf-qmail` > auto_qmail.c'
+./auto-str auto_qmail `head -n 1 conf-qmail`
--- ./default.do.orig	2005-03-15 21:54:27.941456400 -0400
+++ ./default.do	2005-03-15 21:54:27.944455944 -0400
@@ -63,12 +63,12 @@
   auto-ccld.sh)
     dependon conf-cc conf-ld warn-auto.sh
     formake '( cat warn-auto.sh; \'
-    formake 'echo CC=\'\''`head -1 conf-cc`\'\''; \'
-    formake 'echo LD=\'\''`head -1 conf-ld`\'\'' \'
+    formake 'echo CC=\'\''`head -n 1 conf-cc`\'\''; \'
+    formake 'echo LD=\'\''`head -n 1 conf-ld`\'\'' \'
     formake ') > auto-ccld.sh'
     cat warn-auto.sh
-    echo CC=\'`head -1 conf-cc`\'
-    echo LD=\'`head -1 conf-ld`\'
+    echo CC=\'`head -n 1 conf-cc`\'
+    echo LD=\'`head -n 1 conf-ld`\'
     ;;
   *)
     nosuchtarget
--- ./Makefile.orig	2005-03-15 21:54:27.949455184 -0400
+++ ./Makefile	2005-03-15 21:54:27.961453360 -0400
@@ -21,8 +21,8 @@
 auto-ccld.sh: \
 conf-cc conf-ld warn-auto.sh
 	( cat warn-auto.sh; \
-	echo CC=\'`head -1 conf-cc`\'; \
-	echo LD=\'`head -1 conf-ld`\' \
+	echo CC=\'`head -n 1 conf-cc`\'; \
+	echo LD=\'`head -n 1 conf-ld`\' \
 	) > auto-ccld.sh
 
 auto-str: \
@@ -36,7 +36,7 @@
 
 auto_bin.c: \
 auto-str conf-bin
-	./auto-str auto_bin `head -1 conf-bin` > auto_bin.c
+	./auto-str auto_bin `head -n 1 conf-bin` > auto_bin.c
 
 auto_bin.o: \
 compile auto_bin.c
@@ -44,7 +44,7 @@
 
 auto_qmail.c: \
 auto-str conf-qmail
-	./auto-str auto_qmail `head -1 conf-qmail` > auto_qmail.c
+	./auto-str auto_qmail `head -n 1 conf-qmail` > auto_qmail.c
 
 auto_qmail.o: \
 compile auto_qmail.c
@@ -611,8 +611,8 @@
 
 setup: \
 it man install conf-bin conf-man
-	./install "`head -1 conf-bin`" < BIN
-	./install "`head -1 conf-man`" < MAN
+	./install "`head -n 1 conf-bin`" < BIN
+	./install "`head -n 1 conf-man`" < MAN
 
 sgetopt.0: \
 sgetopt.3
--- ./setup.do.orig	2005-03-15 21:54:27.967452448 -0400
+++ ./setup.do	2005-03-15 21:54:27.969452144 -0400
@@ -1,5 +1,5 @@
 dependon it man install conf-bin conf-man BIN MAN
-formake './install "`head -1 conf-bin`" < BIN'
-formake './install "`head -1 conf-man`" < MAN'
-./install "`head -1 conf-bin`" < BIN
-./install "`head -1 conf-man`" < MAN
+formake './install "`head -n 1 conf-bin`" < BIN'
+formake './install "`head -n 1 conf-man`" < MAN'
+./install "`head -n 1 conf-bin`" < BIN
+./install "`head -n 1 conf-man`" < MAN
